docs/releasenotes/release-1.124.md

```diff
@@ -1,24 +1,23 @@
 # v1.124.0
 
-** This version is not yet released; this document is gathering release notes for the future release **
+* Special shout-outs to @600lyy, @acpana, @anhdle-sso, @benjamin-maynard, @cheftako, @gemmahou, @hankfreund, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nancynh, @svetakvsundhar, @xiaoweim, @yuwenma for their contributions to this release.
 
-* ...
+## Announcement
 
-* Special shout-outs to ... for their
-  contributions to this release.
-TODO: list contributors with `git log v1.123.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+* We launched a major improvement to the Config Connector resource development!  Our new approach significantly enhances reliability and provides a more native Kubernetes experience. Learn more in our [guide](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/docs/develop-resources)   
 
-## Resources promoted from alpha to beta:
+* You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on `ComputeForwardingRule` resource to opt-in
+the Direct Cloud Reconciler, which provides sustainable reconciliation support for different types of ForwardingRule uses, including setting labels for ServiceAttachments and Private Service Connect.
 
-*When resources are promoted from alpha to beta, we (generally) ensure they follow our best practices: use of refs on fields where appropriate,
-output fields from GCP APIs are in `status.observedState.*`
+## Resources promoted from alpha to beta:
 
-* `PlaceholderKind`
+* `RedisCluster` (v1beta1)
 
 ## Direct Cloud Reconciler:
```

Comment:
Can we rephrase this?  Maybe we should say something more like:

```
# New Resources

* BigQueryAnalyticsHubDataExchange (v1alpha1, uses direct reconciler model)
```

But I don't know where that leaves the other two resources ...

Reply:
Can we have separate lists for alpha and beta. 

New Enabled By Default Resources:
* `CertificateManagerDNSAuthorization` (uses direct reconciler model)

New Alpha Resources:
* BigQueryAnalyticsHubDataExchange (uses direct reconciler model)

Reply:
If users do not care Alpha resource, how about drop the `Direct Cloud Reconciler` section?